core/roboconf-agent/src/test/java/net/roboconf/agent/internal/Agent_BasicsTest.java

```diff
@@ -225,6 +225,11 @@ public void testReconfigureWhenAgentIsNotStarted() {
 		agent.listPlugins();
 	}
 
+	@Test
+	public void testAgentStatus() {
+
+		Assert.assertNotNull( this.agent.agentStatus() );
```

Comment:
OK. This covers one code branch (the one where there is no message).
How do you verify the other one.

I suggest you complete it with another method where you return a message queue with several messages.
Example...

```java
// Same test but with messages this time.
// Stop the processor.
agent.stop();

// Add messages manually.
client.getMessageProcessor().getMessageQueue().add( new Msg...());
client.getMessageProcessor().getMessageQueue().add( new Msg...());

// Verify these messages are found.
Assert.assertTrue( this.agent.agentStatus().contains( Msg... ));
```